### DISCUSSION TITLE
About Binary Executables' error

### DISCUSSION THREAD

**Discussion by dlyltm** [su2code/SU2#2289] [Q&A]
I run SU2_CFD.exe of Binary Executables, it shows:

Error in "void CConfig::SetConfig_Parsing(char*)":
-------------------------------------------------------------------------
The configuration file (.cfg) is missing!!

Another exe files fail to run ,too. This is true for both serial and parallel versions. What is the reason for this?

**Comment by pcarruscag** [comment]
Please have a look at our documentation and tutorials
https://su2code.github.io/docs_v7/home/

**Reply by dlyltm** [reply]
I followed the method in the Quick Start and obtained corresponding results. But the view of flow.vtu demonstrated by ParaView is different from that on official website. See bellow:
![ParaViewScr](https://github.com/su2code/SU2/assets/33916145/a6c5bb0f-f2aa-43ca-9717-e1aa827bcee4)
Why is this?
